enclave/core/windows/reloc.c

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include <openenclave/internal/globals.h>
+#include <windows.h>
+#include "../init.h"
+
+/*
```

Comment:
Style: use the same block comment format:
```c
/*
 * _oe_reloc_bias is used to calculate relocation difference *BEFORE*
 * relocation. Since _oe_reloc_bias hasn't been relocated yet, it
 * still contains the original value. &_oe_reloc_bias on the other hand,
 * is pc-relative, and is the value *after* relocation. Therefore:
 * 
 *     relocation_diff = (uint64_t)&_oe_reloc_bias - _oe_reloc_bias; 
 */
```

Here and elsewhere in this file and others.

Reply:
fixed.